code/game/objects/items/implants/implantswap.dm

```diff
@@ -0,0 +1,74 @@
+/*!
+The swapper implant comes in two parts, the activator pen, and the swapper implant itself. The activator stores the object reference to the implant, and uses that to swap the locations of the mobs holding the objects.area
+*/
+
+/obj/item/pen/swap_activator
+	///Stores the implant object ref.
+	var/obj/item/implant/swapper/receiver
+
+/obj/item/pen/swap_activator/proc/activate(var/obj/item/implant/swapper/R = receiver, var/obj/A = src)
+	///Grab the recievers mob
+	var/mob/living/RL = R.loc
```

Comment:
Please no simplemobs, I already see this being used to just bring megafauna and other things to the station. Also it being a simplemob really does defeat the purpose of the item in the first place (to be in a place you're not sure what it is).

Reply:
The reason i made it simplemobs is exactly that. I want the uses to be open ended and unrestricted.

Reply:
Yeah but for 5 tc that’s ridiculous how easy that would be.

You could literally get 4 megafauna on the station just by running away from them.

Reply:
This is a pretty massive feature to give the swappers, which could cause unintended balance issues. If it becomes a problem, it's an easy removal though.

Reply:
Don't allow them to swap megafauna please, we already have enough trouble with them already.